examples/next-js/next.config.js

```diff
@@ -0,0 +1,52 @@
+const RollbarSourcemapPlugin = require('rollbar-sourcemap-webpack-plugin');
+
+const nextConfig = {
+  webpack: (config, { dev, webpack }) => {
+    if (!dev) {
+      // generate sourcemaps
+      config.output.futureEmitAssets = false;
+      config.devtool = 'nosources-source-map';
+
+      for (const plugin of config.plugins) {
+        if (plugin.constructor.name === 'UglifyJsPlugin') {
+          plugin.options.sourceMap = true;
+          break;
+        }
+      }
+
+      if (config.optimization && config.optimization.minimizer) {
+        for (const plugin of config.optimization.minimizer) {
+          if (plugin.constructor.name === 'TerserPlugin') {
+            plugin.options.sourceMap = true;
+            break;
+          }
+        }
+      }
+
+      // Generate a common `id` to be used when initializing Rollbar & when uploading the sourcemaps.
```

Comment:
Why not just use the build id provided by Next.js?

Reply:
Same as above, this portion I grabbed from a create-react-app project that we were using the GIT_REVISION in for our code_version

Reply:
Gotcha. I was more just curious, but this definitely works.